plugins/inspect/services_inspector.rb

```diff
@@ -85,4 +97,29 @@ def inspect_sysvinit_services(system)
 
     ServiceList.new(services.sort_by(&:name))
   end
+
+  def inspect_sysvinit_redhat_services(system)
+    system.check_requirement("chkconfig", "--version")
+    system.check_requirement("runlevel")
+
+    output = system.run_command(
+      "chkconfig", "--list",
+      :stdout => :capture
+    )
+
+    previouslevel,runlevel = system.run_command(
```

Comment:
Useless assignment to variable - `previouslevel`. Use `_` or `_previouslevel` as a variable name to indicate that it won't be used.<br>Space missing after comma.